doc/python/imshow.md

```diff
@@ -231,7 +231,7 @@ fig.show()
 
 ### Automatic contrast rescaling in `px.imshow`
 
-When `zmin` and `zmax` are not specified, the `contrast_rescaling` arguments determines how `zmin` and `zmax` are computed. For `contrast_rescaling='minmax'`, the extrema of the data range are used. For `contrast_rescaling='infer'`, a heuristic based on the data type is used:
+When `zmin` and `zmax` are not specified, the `contrast_rescaling` arguments determines how `zmin` and `zmax` are computed. For `contrast_rescaling='minmax'`, the extreme of the data range are used. For `contrast_rescaling='infer'`, a heuristic based on the data type is used:
```

Comment:
this is the only one I disagree with... "extrema" is the correct latin word here although "extremes" would be less confusing to some ;)